gaurun/apns_http2.go

```diff
@@ -54,6 +57,29 @@ func NewApnsClientHttp2(certPath, keyPath string) (*http.Client, error) {
 	}, nil
 }
 
+func loadX509KeyPairWithPassword(certPath, keyPath, keyPassphrase string) (*tls.Certificate, error) {
```

Comment:
The return value of `tls.LoadX509KeyPair` is (`tls.Certificate`, `error`). But what is the reason why the return value of this function is (`*tls.Certificate`, `error`)?

Reply:
Please see https://github.com/golang/go/blob/master/src/crypto/tls/tls.go#L178-L188

In such a case, return (`tls.Certificate{}`, `err`). You do not need to use `nil`.